docs/guide/ecosystem/editors-and-tools.md

```diff
@@ -0,0 +1,92 @@
+- [CodeSandbox](#codesandbox) - in-browser, cloud based, free. 
+
+</useful-links-section>
+</useful-links>
+
+### VSCode <badge text="Hottest"/>
+
+There is no doubt, that VSCode has gained an enormous adoption in the development scene and for good reasons. It is not only highly customisable, with a large selection of plugins, supports a wide range of languages, but its also **free**.
+
+Out of the box, VSCode comes pretty bare bones with all the basic JS, TS and HTML editing tools, so to enable proper Vue support, you will need to install a plugin called `Vetur`. It gives you smart code completion and suggestions, syntax highlighting, type hinting, linting, code snippets and much more. It supports TS, JS, Stylus pretty well, with others to follow soon. You may also install a set of snippets to reduce typing the same boilerplate.
+
+Vetur is being developed closely with VSCode and Vue core teams, to ensure maximum parity.
+
+<useful-links>
+<useful-links-section title="Useful Links">
+
+- [Official Website](https://code.visualstudio.com/)
+- [Vetur Plugin](https://marketplace.visualstudio.com/items?itemName=octref.vetur) - enable improved Vue support
+- [Vue VSCode Extension Pack](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-extensionpack) - collection of useful plugins for Vue app development
+
+</useful-links-section>
+</useful-links>
+
+### WebStorm/PhpStorm
+
+If you need a fully featured IDE with all the bells and whistles, not just a code editor, WebStorm is a serious contender. Not free, JetBrains products like WebStorm and PhpStorm are feature rich IDEs, offering everything a web developer may need for their daily work. They have a free trial, as well as a beta testing program, which allows users to test JetBrains products for free.
+
+To enable Vue support you will need to install the official, free Vue.js plugin from the plugins menu. It will give you `.vue` file support with highlighting, `template` and `script` tag code completion, linting, component hinting, prop completion, type hinting, refactoring support and much more.
+
+As a fully featured IDE, WebStorm provides greater refactoring capabilities, improved code completion, hinting and highlighting, but at a cost of speed. It is heavy and can get choppy if you open too many files or projects at the same time.
+
+### CodeSandbox <badge text="Rising Star"/>
+
+CodeSandbox is the only online, cloud based code editor, that supports Vue's SFC format, among many other frameworks and popular client apps and server setups. Vue projects are built with Vue CLI, giving you a well known folder structure to extend upon. 
+
+Based on VSCode, CodeSandbox has a similar look and feel. When you create a Vue project, you will benefit from code completion, hinting, syntax highlighting and more. Apart from Vue CLI, they have templates for [Vuepress](../ecosystem/documentation.md#vuepress) and [Nuxt](../ecosystem/nuxt.md), just to name a few.
```

Comment:
I'd add somewhere that we can integrate Codesandbox with Github (or Git in overall?). Imho that's the main feature that makes it fit this category at all.